dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -600,8 +600,16 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         auto args = winrt::make<winrt::Microsoft::Windows::PushNotifications::implementation::PushNotificationReceivedEventArgs>(payload, length);
 
         auto lock{ m_lock.lock_exclusive() };
```

Comment:
Curious but why do we have the argument foregroundHandled at all in this function? Shouldn't it be removed?

Reply:
This is for unpackaged scenarios attempting to activate a foreground handler. If no handlers are registered, it will protocol launch a new instance.

Reply:
Or in the packaged treated as unpackaged, CoCreateInstance.

Reply:
Can you point me to where in our code is *foregroundHandled being used to do this?

Reply:
Is it this?
PushNotificationManager::OnRawNotificationReceived

Reply:
I don't see any CoCreateInstance for packaged applications being done here.

Reply:
NotificationListener::OnRawNotificationReceived in the Long Running Process

Reply:
ForegroundSinkManager::InvokeForegroundHandlers

Reply:
What about packaged apps that are getting activated through COM? 
PushNotificationManager::OnRawNotificationReceived doesnt do  anything to account for them and launching a new process.

Reply:
OnRawNotificationReceived is only activated if a NotificationSink is registered. Sink registration should be wrapped in a foreground handler check. I added this in my PR to force BI to activate the packaged app.

https://github.com/microsoft/WindowsAppSDK/pull/2187